### PR TITLE
#121 fix for voice call not working on latest sdk

### PR DESCRIPTION
**Pull request by tanmoy27112000**

### CODE REVIEW

File: android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java [hunk 1]
```diff
@@ -200,6 +200,106 @@ public void onClick(View v) {
 
                     }
                 });
+                btn1.setOnClickListener(new View.OnClickListener() {
```
Comment (cybex-dev) [inline android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java]: Do you think we can condense this, a bit too much repetition.

File: android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java [hunk 2]
```diff
@@ -166,6 +166,41 @@ public void onClick(View v) {
...
+                    @Override
+                    public void onClick(View v) {
+                        String currentText = keypadView.getText().toString();
+                        if (!currentText.isEmpty()) {
+                            keypadView.setText(currentText.substring(0, currentText.length() - 2));
+                        }
+                    }
+                });
+
+                btn0.setOnClickListener(new View.OnClickListener() {
```
Comment (cybex-dev) [inline android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java]: Let's maybe reduce code duplication here too

**Comment by cybex-dev** [comment]
Thanks for the great feature addition. If you could add a screenshot of the feature addition, that'd be great.

Before merging changes, please:
1. Do a fetch & merge
2. Please review your PR for unnecessary code additions, metadata files, etc such as additional new lines, etc. including only necessary changes to add the feature.

File: android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java (hunk 1 above)
Comment (tanmoy27112000) [reply android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java]: Hi, I am very new to native android development, any input from your side will help

**Pull request closed without merging**